core/clients/key_flow.go

```diff
@@ -99,14 +99,6 @@ func (c *KeyFlow) GetServiceAccountEmail() string {
 	return c.key.Credentials.Iss
 }
 
-// GetToken returns the token field
```

Comment:
This method is used in the CLI for the keyFlow

Reply:
We can't remove it, unless we would export the `token` field. But in this case I think it's not worth it to make the breaking change. I'm not sure why I didn't just make the field public at that time (I thought I did :thinking: )